src/back-end/products-db.js

```diff
@@ -124,9 +124,7 @@ const productsList =
         "rating":3.7,
         "price":7000,
         "category":"bangles",  
-    },
-   
-    
+    },   
 ]
```

Comment:
could try adding a product for the 5000 slab of the price range filter so it doesnt show up an empty product section.

Reply:
yeah! will do that